### DISCUSSION TITLE
诊断模块如何发出流控帧?

### DISCUSSION THREAD

**Discussion by Mkiller0420** [TOSUN-Shanghai/TSMaster#97] [Q&A]
Hello, I sent a 22 request through the Diagnostic Console during the use of the diagnostic module, and the ECU replied with a multi-frame first frame. However, TSMaster did not send a flow control frame, which caused the diagnostic session to be interrupted. My settings are shown in the figure, how can I make TSMaster send a flow control frame?

![image](https://user-images.githubusercontent.com/31810823/224983024-acfd0ad8-e381-4cf5-8384-9778e1ec0321.png)

![image](https://user-images.githubusercontent.com/31810823/224983597-bead9252-b89f-4c15-acb2-8f32ddb010f8.png)

![image](https://user-images.githubusercontent.com/31810823/224983200-67868726-05a2-415e-998b-53675d698f5c.png)

![image](https://user-images.githubusercontent.com/31810823/224983290-fed9e3d3-3409-44d1-a913-0424e7c545bb.png)

**Comment by totocaca123** [comment]
I triés to send a program flow with c script. It works but diagnostic console does not sée next frames. 
This feature seems blocked unless ypu use a tosun hardware (i tried with peak or vector and still dame issue)
